src/pages/workspaces/workspace/Data.js

```diff
@@ -447,6 +447,18 @@ const WorkspaceData = _.flow(
 
   const toSortedPairs = _.flow(_.toPairs, _.sortBy(_.first))
 
+  const getRunningImportJobs = async () => {
+    try {
+      const runningJobs = await Ajax(signal).Workspaces.workspace(namespace, name).listImportJobs(true)
+      _.map(job => {
+        const jobStatus = _.lowerCase(job.status)
+        if (!(jobStatus === "success" || jobStatus === "error" || jobStatus === "done"))
```

Comment:
I think this is a place where a bit more concision might help readability:
```suggestion
        if (!_.includes(jobStatus, ['success', 'error', 'done']))
```